kotlin/workflow-core/src/main/java/com/squareup/workflow/WorkflowAction.kt

```diff
@@ -100,5 +101,7 @@ interface WorkflowAction<StateT, out OutputT : Any> : (StateT) -> Pair<StateT, O
       output: OutputT
     ): WorkflowAction<StateT, OutputT> =
       WorkflowAction({ "emitOutput($name, $output)" }) { Pair(it, output) }
+
+    private val NO_ACTION = WorkflowAction<Any, Any>({ "noop" }) { Pair(it, null) }
```

Comment:
Nit: Oops, I forgot to change this to "noAction" – would you mind doing that here right away? But it's not a regression, so I'm also fine merging this as-is.

Reply:
👍